spec/support/capybara.rb

```diff
@@ -1,8 +1,28 @@
 require "capybara/rails"
 require "capybara/rspec"
+require "selenium-webdriver"
+
+Capybara.register_driver(:remote_firefox) do |app|
+  url = "http://selenium:4444/wd/hub"
+  options = Selenium::WebDriver::Firefox::Options.new
+  options.add_argument("--headless")
```

Comment:
selenium-webdriverのDocコメントにあるように、簡単な書き方ある。

ちなみに、`--headless`だと間違いで、正しくは`-headless`。
こういう間違いを防ぐためにも、用意されているメソッドは使うべき。

```suggestion
  options.headless!
```

Reply:
Fixed at d48f24dc597cc79bb6c6e98362f296b12c0346eb